### PR TITLE
Add swimming_pool

### PR DESCRIPTION
**Pull request by rmarianski**
Refs #443

### CODE REVIEW

File: data/functions.sql [hunk 1]
```diff
@@ -371,13 +371,16 @@ END;
...
-    waterway_val text, natural_val text, landuse_val text)
+    amenity_val text, landuse_val text, leisure_val text,
+    natural_val text, waterway_val text)
 RETURNS BOOLEAN AS $$
 BEGIN
     RETURN (
         waterway_val IN ('riverbank', 'dock')
      OR natural_val IN ('water')
      OR landuse_val IN ('basin', 'reservoir')
+     OR amenity_val='swimming_pool'
```
Comment (nvkelso) [inline data/functions.sql]: We usually say `OR amenity_val IN ('swimming_pool')`, is there a performance reason not to do that here?

**Comment by nvkelso** [comment]
Before / after images?

File: data/functions.sql (hunk 1 above)
Comment (zerebubuth) [inline data/functions.sql]: I don't think there's a significant performance difference.

**Comment by nvkelso** [comment]
LGTM, let's merge this.

**Pull request merged**